lib/pharos/phases/configure_custom_network.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    class ConfigureCustomNetwork < Pharos::Phase
+      title "Configure Custom network"
+
+      def call
+        logger.info { "Configuring custom network ..." }
+        # TODO Set options more granularly?
+        stack = Pharos::Kube.stack('custom-network', @config.network.custom.manifest_path, name: 'custom_network', cluster_config: @config)
```

Comment:
@jakolehm @kke WDYT, should the options be set "directly" or is it enough that they can be accessed in the templates through the cluster level config struct?

Reply:
IMO `cluster_config` is enough for now.